src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertFromJsonCommand.cs

```diff
@@ -43,6 +42,13 @@ public class ConvertFromJsonCommand : Cmdlet
         [ValidateRange(ValidateRangeKind.Positive)]
         public int Depth { get; set; } = 1024;
 
+        /// <summary>
+        /// Gets or sets the switch to prevent ConvertFrom-Json from unravelling collections during deserialization, instead passing them as a single
+        /// object through the pipeline.
+        /// </summary>
+        [Parameter]
+        public SwitchParameter NoEnumerate { get; set; }
```

Comment:
Please see updated Committee review here https://github.com/PowerShell/PowerShell/issues/3424#issuecomment-548141395

Reply:
@PowerShell/powershell-committee re-reviewed this and agree with this proposed change (not a statement on the code review)

Reply:
So nothing to do here, right?